core/foundation/src/TClassEdit.cxx

```diff
@@ -1965,6 +1965,10 @@ class NameCleanerForIO {
       // Perhaps we could treat atomics as well like this?
       if (!fMother && TClassEdit::IsUniquePtr(fName+"<")) {
          name = fArgumentNodes.front()->ToString();
+         if (name.find("const ") == 0) {
```

Comment:
What about ("TH1F const" and "TH1F const*")?  [Should we be using something like TSplitType)?

Reply:
but this is the template of a unique_ptr, does your remark apply?

Reply:
Maybe ... maybe not.   I was think we ought to handle properly also the cases:
```
unique_ptr<Type const>
unique_ptr<Type const*> // yes ought to be rare.
```